inc/dbmysql.class.php

```diff
@@ -243,16 +222,13 @@ function prepare($query) {
    }
 
    /**
-    * Give result from a mysql result
-    *
-    * @param $result    MySQL result handler
-    * @param $i         Row to give
-    * @param $field     Field to give
-    *
-    * @return Value of the Row $i and the Field $field of the Mysql $result
-   **/
+    * Give result from a sql result
```

Comment:
+1 
https://phpdoc.org/docs/latest/getting-started/your-first-set-of-documentation.html#example

Maybe we should create a § on readthedocs ?